javalib/src/main/scala/java/security/AccessController.scala

```diff
@@ -0,0 +1,8 @@
+package java.security
+
+// Ported from Harmony
+
+object AccessController {
+  def doPrivileged[T](action: PrivilegedAction[T]): T =
+    action.run()
+}
```

Comment:
We're not going to support access control APIs from JDK. Please remove `AccessController` and `PrivilegedAction` and replace all of its uses as if the privilege is always granted.